docs/source/reservations/reservations.rst

```diff
@@ -170,32 +170,18 @@ In case you want to specify the DWS configuration for each job/cluster, you can
 
 .. _dws-on-gke:
 
-Using DWS on GKE with Kueue
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Using DWS on GKE
+~~~~~~~~~~~~~~~~~
 
-DWS is also supported on Google Kubernetes Engine (GKE) with Kueue. To enable DWS on GKE, you need to set up your GKE cluster with Kueue and DWS; see the `GCP official docs <https://cloud.google.com/kubernetes-engine/docs/how-to/provisioningrequest>`__.
+GKE supports the two kinds of flex-start configurations:
 
-To launch a SkyPilot cluster or job on GKE with DWS, you can specify the DWS configuration in the SkyPilot task YAML:
+* Flex-start, where GKE allocates resources node by node. This configuration only requires you to set the `--flex-start` flag during node creation.
+* Flex-start with queued provisioning, where GKE allocates all requested resources at the same time. To use this configuration, you have to add the `--flex-start` and `enable-queued-provisioning` flags when you create the node pool. 
```

Comment:
mention some examples for when each mode is best suited. Also, considering SkyPilot clusters are always gang-scheduled, it does not make sense to use `flex-start` with `kueue` setup, right?

Reply:
Updated